core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -474,6 +458,21 @@
       PropertyType.TIMEDURATION,
       "The maximum amount of time to wait after a failure to create or write a write-ahead log.",
       "1.7.1"),
+  // Crypto-related properties
+  @Experimental
+  TSERV_WAL_CRYPTO_PREFIX("tserver.wal.crypto.opts.", null, PropertyType.PREFIX,
+      "Properties related to on-disk file encryption.", "2.1.0"),
+  @Experimental
+  @Sensitive
+  TSERV_WAL_CRYPTO_SENSITIVE_PREFIX("tserver.wal.crypto.opts.sensitive.", null, PropertyType.PREFIX,
+      "Sensitive properties related to on-disk file encryption.", "2.1.0"),
+  @Experimental
+  TSERV_WAL_CRYPTO_SERVICE("tserver.wal.crypto.service",
+      "org.apache.accumulo.core.spi.crypto.NoCryptoService", PropertyType.CLASSNAME,
+      "The class which executes on-disk write ahead log encryption/decryption. The default does "
+          + "nothing. This property must be a classname with an implementation of the "
+          + "org.apache.accumulo.core.spi.crypto.CryptoService interface.",
+      "2.1.0"),
```

Comment:
I would expect all tservers to use the same WAL options, so that files written by one tserver will be able to be read by another. This implies the use of `instance.` properties for WAL crypto options, not `tserver.` Also, if a crypto service factory is specified in an `instance.` property, then there's no need to specify a specific crypto service here, as the factory would make that decision based on the options and scope.

Reply:
I will try out both types of configuration using `instance` properties.